packages/onboarding-ui/src/pages/CreateCloudWorkspacePage/DescriptionIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import type { ReactElement } from 'react';
+
+const DescriptionIcon = (): ReactElement => (
+  <svg
+    width='12'
+    height='12'
+    viewBox='0 0 15 15'
+    fill='#1ECB92'
+    xmlns='http://www.w3.org/2000/svg'
+  >
+    <g clip-path='url(#clip0)'>
```

Comment:
There is no clip path in this SVG document. I believe it's safe to remove this `<g>` parent element.